README.md

```diff
@@ -1,2 +1,3 @@
 # directory-negotiator-sync
+
 Service to synchronize ressources from the BBMRI-ERIC Directory into the BBMRI-ERIC Negotiator.
```

Comment:
add a full description with a table of variables and their description

Reply:
Fixed in https://github.com/BBMRI-ERIC/directory-negotiator-sync/pull/1/commits/c8de481640bc91cf89dfde9dc73e0778f905dcae